src/test/frank-single-transaction.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# bash strict mode
+set -euo pipefail
+IFS=$'\n\t'
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "${SCRIPT_DIR}/../../"
+
+# create test configuration for fddev
+TMPDIR=$(mktemp -d)
+cat > ${TMPDIR}/config.toml <<EOM
+[development]
+    sudo = true
+    sandbox = true
+    [development.netns]
+        enabled = true
+[tiles.quic]
+    interface = "veth_test_xdp_0"
+[layout]
+    affinity = "0-9"
+    verify_tile_count = 2
+    bank_tile_count = 2
+EOM
+export FIREDANCER_CONFIG_TOML=${TMPDIR}/config.toml
+
+# acquire and initialize everything to build
+echo y | ./deps.sh
```

Comment:
Can use richies composite deps.sh step instead of having this in here (I think it's cleaner for a few reasons, another one being you can run this on your checkout without it kinda overwriting things)

Reply:
Addressed in https://github.com/firedancer-io/firedancer/pull/586 - In the future I would like to have a script we can run after a clean checkout that handles all the dependencies, building, and does a smoke test.